### ISSUE TITLE
Getting All Good (0 modules) instead of errors

### ISSUE THREAD

**Issue by jjoekoullas** [ndmitchell/ghcid#73]
On Windows.
Installed via: `cabal install ghcid`
Run via: `ghcid --command="stack ghci"` in a stack project
Inserting an error and saving causes the ghcid window to refresh and just display "All Good (0 modules)".  

Here's the -v output with an error inserted:
```
C:\Users\jjoed\work\training\stuff [master +1 ~2 -0 !]> ghcid --command="stack
 ghci" -v
Loading stack ghci ...
%GHCINP:
%GHCERR: stuff-0.1.0.0: build
stuff-0.1.0.0: build
%GHCERR: Preprocessing library stuff-0.1.0.0...
Preprocessing library stuff-0.1.0.0...
%GHCERR: [1 of 1] Compiling Main             ( src\Main.hs, .stack-work\dist\2672c1f3\build\Main.o )
[1 of 1] Compiling Main             ( src\Main.hs, .stack-work\dist\2672c1f3\build\Main.o )
%GHCERR:

%GHCERR: C:\Users\jjoed\work\training\stuff\src\Main.hs:32:5:
C:\Users\jjoed\work\training\stuff\src\Main.hs:32:5:
%GHCERR:     Couldn't match expected type `Int' with actual type `[Char]'
    Couldn't match expected type `Int' with actual type `[Char]'
%GHCERR:     In the expression: ""
    In the expression: ""
%GHCERR:     In an equation for `x': x = ""
    In an equation for `x': x = ""
%GHCERR:

%GHCERR: --  While building package stuff-0.1.0.0 using:
--  While building package stuff-0.1.0.0 using:
%GHCERR:       C:\Users\jjoed\AppData\Roaming\stack\setup-exe-cache\x86_64-windows\setup-Simple-Cabal-1.22.5.0-ghc-7.10.3.exe --builddir=.stack-work\dist\2672c1f3 build lib:stuff exe:stuff --ghc-options " -ddump-hi -ddump-to-file"
      C:\Users\jjoed\AppData\Roaming\stack\setup-exe-cache\x86_64-windows\setup-Simple-Cabal-1.22.5.0-ghc-7.10.3.exe --builddir=.stack-work\dist\2672c1f3 build lib:stuff exe:stuff --ghc-options " -ddump-hi -ddump-to-file"
%GHCERR:     Process exited with code: ExitFailure 1
    Process exited with code: ExitFailure 1
%GHCERR: Warning: build failed, but optimistically launching GHCi anyway
Warning: build failed, but optimistically launching GHCi anyway
%GHCERR: Using main module: 1. Package `stuff' component exe:stuff with main-is file: C:\Users\jjoed\work\training\stuff\src\Main.hs
Using main module: 1. Package `stuff' component exe:stuff with main-is file: C:\Users\jjoed\work\training\stuff\src\Main.hs
%GHCERR: Warning: Omitting load of module Main, because it matches the filepath of the Main target, C:\Users\jjoed\work\training\stuff\src\Main.hs
Warning: Omitting load of module Main, because it matches the filepath of the Main target, C:\Users\jjoed\work\training\stuff\src\Main.hs
%GHCERR: Configuring GHCi with the following packages: stuff
Configuring GHCi with the following packages: stuff
%GHCOUT: GHCi, version 7.10.3: http://www.haskell.org/ghc/  :? for help
GHCi, version 7.10.3: http://www.haskell.org/ghc/  :? for help
%GHCOUT: Ok, modules loaded: none.
Ok, modules loaded: none.
%GHCOUT: [1 of 1] Compiling Main             ( C:\Users\jjoed\work\training\stuff\src\Main.hs, interpreted )
[1 of 1] Compiling Main             ( C:\Users\jjoed\work\training\stuff\src\Main.hs, interpreted )
%GHCERR:

%GHCOUT: Failed, modules loaded: none.
%GHCERR: C:\Users\jjoed\work\training\stuff\src\Main.hs:32:5:
Failed, modules loaded: none.
C:\Users\jjoed\work\training\stuff\src\Main.hs:32:5:
%GHCERR:     Couldn't match expected type `Int' with actual type `[Char]'
%GHCOUT: Prelude> #~GHCID-START~##~GHCID-START~##~GHCID-START~##~GHCID-FINISH~#
    Couldn't match expected type `Int' with actual type `[Char]'
%GHCERR:     In the expression: ""
    In the expression: ""
%GHCERR:     In an equation for `x': x = ""
    In an equation for `x': x = ""
%GHCERR: *** Exception: #~GHCID-FINISH~#
%GHCINP: :show modules
%GHCOUT: #~GHCID-START~##~GHCID-START~##~GHCID-START~##~GHCID-FINISH~#
%GHCERR: *** Exception: #~GHCID-FINISH~#
%MESSAGES: [Loading {loadModule = "Main", loadFile = "C:\\Users\\jjoed\\work\\training\\stuff\\src\\Main.hs"},Loading {loadModule = "Main", loadFile = "src\\Main.hs"}]
%LOADED: []
%CONSOLE: width = 78, height = 49

All good (0 modules)















































%WAITING: C:\Users\jjoed\work\training\stuff\src\Main.hs src\Main.hs
%WAITING: C:\Users\jjoed\work\training\stuff\src
```

**Comment by ndmitchell** [comment]
Thanks for the report. The trace suggests its caused by #61, which got fixed in 0.6.2. Which version of ghcid are you running?

**Comment by jjoekoullas** [comment]
You're correct, this was due to my having an old version.

I had forgotten that I had `stack install ghcid`'d, which put 5.4 on my local path.  I tried to `stack update` and `stack install` again, since I saw the stackage badge lists 0.6.4 but to no avail, it was still installing the old version. I'm not very knowledgeable about how to use stack, so it could just be that I didn't know how to clear out a cached version.  `cabal install` was grabbing the correct version, so I just dropped it into my path and everything's working now.

Thanks for the help, sorry for the noise.
